### ISSUE TITLE
[Bug] Parsing error. v1.2.0

### ISSUE THREAD

**Issue by jsmankoo** [mransan/ocaml-protoc#130]
### Expectation
Successfully parses proto file and creates appropriate ocaml files

### Actual
```
○ → ocaml-protoc -bs -binary -ml_out src ./test.proto
./test.proto:6:0: Parsing error.
```

<img width="451" alt="screen shot 2018-09-30 at 4 28 48 pm" src="https://user-images.githubusercontent.com/6406020/46262350-f3a7cb80-c4cd-11e8-8512-278f413c5a4b.png">

**Comment by mransan** [comment]
Good catch, need to add support for enum options.

**Comment by mransan** [comment]
Fixed in #145 will start an opam release.
